tests/integration/workflows/test_manual_merge_workflow.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import pytest
+
+from invenio_workflows import ObjectStatus, workflow_object_class
+
+from inspirehep.modules.workflows.tasks.merging import read_wf_record_source
+from inspirehep.modules.workflows.workflows.manual_merge import start_merger
+from inspirehep.utils.record import get_source
+from inspirehep.utils.record_getter import get_db_record, RecordGetterError
+
+from calls import do_resolve_manual_merge_wf
+
+
+def fake_record(title, rec_id):
+    return {
+        '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'titles':  [{'title': title}],
+        '_collections': ['Literature'],
+        'document_type': ['article'],
+        'acquisition_source': {'source': 'arxiv'},
+        'arxiv_eprints': [{'value': '1701.01431', 'categories': ['cs']}],
+        'control_number': rec_id
+    }
+
+
+def test_manual_merge_existing_records(workflow_app):
+    # celery task, to import locally
+    from inspirehep.modules.migrator.tasks import record_insert_or_replace
```

Comment:
Are you sure we need this internal import? If you're not executing this with `delay` it doesn't go to Celery... For example https://github.com/inspirehep/inspire-next/blob/d711dd70d35944e947195058712a73d502752730/tests/integration/refextract/test_refextract_tasks.py#L32